src/common/util.c

```diff
@@ -229,3 +230,30 @@ char *oscap_expand_ipv6(const char *input)
 
 	return ret;
 }
+
+char *oscap_path_join(const char *path1, const char *path2)
+{
+	if (path1 == NULL) {
+		return oscap_strdup(path2);
```

Comment:
path2 can also be NULL at this point, right?

Reply:
yes, but oscap_strdup checks if its argument is NULL

Reply:
ah, ok